app/models/concerns/team_import.rb

```diff
@@ -4,33 +4,33 @@
 module TeamImport
```

Comment:
Why you commented just some lines and not the whole file? Are the uncommented lines used for anything else that is not related to the `TeamImport` feature?